content/influxdb3/enterprise/reference/config-options.md

```diff
@@ -844,36 +855,25 @@ Specifies the size of the RAM cache used to store data, in bytes.
 
 #### exec-mem-pool-bytes
 
-Specifies the size of the memory pool used during query execution, in bytes.
+Specifies the size of memory pool used during query execution.
+Can be given as absolute value in bytes or as a percentage of the total available memory (for example: `10%`).
 
-**Default:** `8589934592`
+**Default:** `20%`
 
 | influxdb3 serve option  | Environment variable            |
 | :---------------------- | :------------------------------ |
 | `--exec-mem-pool-bytes` | `INFLUXDB3_EXEC_MEM_POOL_BYTES` |
 
 ---
 
-#### buffer-mem-limit-mb
-
-Specifies the size limit of the buffered data in MB. If this limit is exceeded,
-the server forces a snapshot.
-
-**Default:** `5000`
-
-| influxdb3 serve option  | Environment variable            |
-| :---------------------- | :------------------------------ |
-| `--buffer-mem-limit-mb` | `INFLUXDB3_BUFFER_MEM_LIMIT_MB` |
-
----
-
 #### force-snapshot-mem-threshold
+<span id="buffer-mem-limit-mb" />
```

Comment:
I assume this span is here to act as a link anchor, but why? Did the configuration name change?

Reply:
span is the old option name. I added all the spans to prevent breaking existing links. I know we don't usually, but didn't want users to be **as** confused